config/tests/fixtures/empty_ok_genesis.json

```diff
@@ -0,0 +1,4 @@
+{
+  "transactions": [],
+  "executor": "./executor.wasm"
```

Comment:
again an inconsistency. We have `genesis_file`, but it's not `executor_file`

Reply:
Agree

Reply:
I propose to change it separately.